CommonWallet/Core/Classes/Common/Model/WalletHistoryRequest.swift

```diff
@@ -9,7 +9,6 @@ import Foundation
 public struct WalletHistoryRequest: Codable, Equatable {
     
     public var assets: [String]?
-    public var filter: String?
```

Comment:
Let's return the field. It is not used by Capital itself but needed when one has custom Filter screen and there is a need to provide custom filters to backend